frontend/src/pages/pipelines/global/modelCustomization/utils.ts

```diff
@@ -163,3 +171,40 @@ export const translateIlabFormToBaseModelInput = (
   sdg_base_model: data.baseModel.sdgBaseModel,
   // TODO more output model fields
 });
+
+export const translateIlabFormToRunType = (
+  data: ModelCustomizationFormData,
+): {
+  runType: string;
+} => ({
+  runType: data.runType.value,
```

Comment:
this should be sdg_pipeline see docs https://github.com/opendatahub-io/ilab-on-ocp/tree/main?tab=readme-ov-file#available-pipeline-parameters

also update the enum values